api/routes/authRouter.js

```diff
@@ -0,0 +1,124 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcryptjs");
+
+const db = require("../../data/dbConfig");
+const { generateToken } = require("../../auth/authenticate");
+
+const router = express.Router();
+
+const secret =
+  process.env.JWT_SECRET ||
+  "add a .env file to root of project with the JWT_SECRET variable";
+
+// const generateToken = user => {
+//   const payload = {
+//     username: user.username
+//   };
+//   const options = {
+//     expiresIn: "1d",
+//     jwtid: 12345
+//   };
+//   return jwt.sign(payload, secret, options);
+// };
+
+router.post("/register", () => {
+  const user = req.body;
+
```

Comment:
Your overall backend codebases structure looks good Angela 👍 Well done handling edge cases and sanitizing user inputs.